edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -79,12 +79,12 @@ <h2 id="acessible-error-title">{% trans "Cannot Start Proctored Exam" %}</h2>
 </div>
 
 <div class="footer-sequence border-b-0 padding-b-0">
-  {% if not is_sample_attempt %}
-  <p class="proctored-exam-instruction">
+  {% if not is_sample_attempt and allow_unproctored %}
+    <p class="proctored-exam-instruction">
       {% trans "Don't want to take this exam with online proctoring?" %}<a class="proctored-decline-exam" href='#' data-action="decline" data-exam-id="{{exam_id}}" data-change-state-url="{{change_state_url}}">
         {% trans "Take this exam as an open exam instead." %}
```

Comment:
I can't run through the workflow on the sandbox, so I can't verify which templates are used. Consolidation/clean up would be great; it seems that we might have either outdated or unused templates? Wording does not match the screenshot of the template that's currently in the Learner documentation: http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_proctored_exams.html#agree-to-take-the-proctored-exam